lib/Camp/Master.pm

```diff
@@ -2393,7 +2393,9 @@ EOF
 sub _initialize_camp_database_mysql {
     my $conf = shift;
     _render_database_config($conf)  unless ($conf->{_did_render_database_config});
-    my $cmd = "mysql_install_db --datadir=$conf->{db_data} --defaults-file=$conf->{db_conf}";
+    my $cmd  = (db_version_mysql() >= 5.7) ?
+      "mysqld  --defaults-file=$conf->{db_conf} --initialize-insecure --datadir=$conf->{db_data}" :
```

Comment:
@hexfusion Is the order of the command line arguments important? If not, I think they should be the same order between the two versions so it's obvious what is different between the two.

Reply:
@jonjensen yeah order matters here sorry I missed this question.